### ISSUE TITLE
Screen reader does not announce the appropriate message when activating the full screen mode

### ISSUE THREAD

**Issue by nightpig** [videojs/video.js#6437]
## Description
When activating the full screen mode, screen reader (VoiceOver/Safari) does not announce the appropriate message for letting users know full screen mode is on.

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. Go https://videojs.com/advanced?video=disneys-oceans by Safari
2. Navigate to 'full screen' button and activate it by VoiceOver

### Expected
We expect there is a message for letting users know full screen is on.
For example, in youtube.com, VoiceOver announces "YouTube Video Player in fullscreen".

### Actual
VoiceOver only announces "Dialog" and "You are currently at Dialog".

## Additional Information
Not sure if we can implement it via some APIs of video.js.
Any suggestions will be greatly appreciated.

**Comment by gkatsev** [comment]
That's interesting. Makes sense to have but I'm not sure that Video.js is currently set up to notify SR users. From what I understand, to have a SR "alert" the element needs to be on the page during player creation, so, it will likely require a new feature in Video.js for providing such an element. Will require some more thought on that.
